src/components/item-info/index.js

```diff
@@ -31,19 +39,12 @@ export const ItemInfo = ({
   const [showSignStatus, setShowSignStatus] = useState(false)
 
   if (isDetailView) {
-    // subtract burned pieces from total
-    let total = 0
-
-    total = supply
-    let ed =
-      token_holders.filter(
-        (e) => e.holder_id === 'KT1HbQepzV1nVGg8QVznG7z4RcHseD5kwqBn'
-      ).length > 0
-        ? token_holders.filter(
-          (e) => e.holder_id === 'KT1HbQepzV1nVGg8QVznG7z4RcHseD5kwqBn'
-        )[0].quantity
-        : 'X'
-    swaps = swaps.filter(e => parseInt(e.contract_version) === 2 && parseInt(e.status) === 0 && e.is_valid)
+    // TODO: subtract burned pieces from total
```

Comment:
That was also not taken into account before, right?

Reply:
nopes. i think the original comment was supposed to be a todo. so i just added 'TODO: ' in front :)

Reply:
Indexer update token supply when some editions are transfered to burn address, so I guess it's more some old logic code coming from before hicdex as there is no need to substract anything.